### PR TITLE
Update Fulcio helm chart for prerelease

### PR DESCRIPTION
**Pull request by haydentherapper**
Signed-off-by: Hayden Blauzvern <hblauzvern@google.com>

## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/fulcio/Chart.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ description: |
 
 type: application
 
-version: 0.5.2
-appVersion: 0.6.0
+version: 0.9.9
```
Comment (haydentherapper) [inline charts/fulcio/Chart.yaml:8]: Happy to change if there's a preference, I just thought we should signify this is approaching 1.0

Comment (hectorj2f) [reply charts/fulcio/Chart.yaml:8]: It is a bit risky in case we need to update the chart with a new release candidate in case of bugs before 1.0

Comment (haydentherapper) [reply charts/fulcio/Chart.yaml:8]: I was thinking we could just add .9 again for each rc. Happy to revert this back though and just increment the minor version until 1.0, no strong preference!

Comment (hectorj2f) [reply charts/fulcio/Chart.yaml:8]: It is okay with me, just wanted to know how we'll deal with more RCS.

**Review by hectorj2f — DISMISSED** [review]

File: charts/fulcio/Chart.yaml (hunk 1 above)
Comment (hectorj2f) [inline charts/fulcio/Chart.yaml:8]: I had a look and I believe we could create a Chart with `version: 1.0.0-rc.0`, cc @cpanato

Comment (haydentherapper) [reply charts/fulcio/Chart.yaml]: Updated!

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**